### PR TITLE
Update docs for wip-catalog chart

### PR DESCRIPTION
**Pull request by pmorie**
Add instructions for the wip-catalog chart.

### CODE REVIEW

File: deploy/wip-catalog/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+### What to deploy, and from where
+
+- `registry`  (optional): Container registry with Service Catalog images; may
+  be left blank if deploying on a local cluster
+- `version`   (optional): Version of Service Catalog (container images) to deploy
+- `namespace` (optional): A Kubernetes namespace to use for Service Catalog deployment
+
+### Configuration of the catalog
+
+- `storageType` (optional): The type of storage for the API server to use
```
Comment (krancour) [inline deploy/wip-catalog/README.md]: Enumerate acceptable values?

File: deploy/wip-catalog/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- `verbosity` (optional): The verbosity of logs
+- `debug` (optional): Whether to create a load balancer for the apiserver and
```
Comment (krancour) [inline deploy/wip-catalog/README.md]: Enumerate acceptable values? `true` to enable? Or `1`?

File: deploy/wip-catalog/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+- `registry`  (optional): Container registry with Service Catalog images; may
+  be left blank if deploying on a local cluster
+- `version`   (optional): Version of Service Catalog (container images) to deploy
+- `namespace` (optional): A Kubernetes namespace to use for Service Catalog deployment
+
+### Configuration of the catalog
+
+- `storageType` (optional): The type of storage for the API server to use
+- `verbosity` (optional): The verbosity of logs
```
Comment (krancour) [inline deploy/wip-catalog/README.md]: Accepted value? Numeric? Or the familiar "info", "debug", "warn", etc.? Upper or lowercase? Or does it not matter?

File: deploy/wip-catalog/templates/apiserver.yaml [hunk 4]
```diff
@@ -68,7 +68,7 @@ spec:
     nodePort: {{ default 30001 .Values.insecureServicePort }}
     port: 80
     targetPort: {{ default 8081 .Values.insecurePort }}
-  {{- end }}
+  {{ end }}
```
Comment (krancour) [inline deploy/wip-catalog/templates/apiserver.yaml]: Why is that being removed? I think the result's going to be an extra line break, isn't it?

**Comment by pmorie** [comment]
I've decided to narrow the scope of this PR to just updating the docs for the `wip-catalog` chart.

File: deploy/wip-catalog/README.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    the cluster if deploying to a non-local cluster. Example (run from this
+    directory):
+
+    ```
+    $ export REGISTRY=hub.docker.com/<username>
+    $ make -C ../.. push
+    ```
+
+    __Note:__ If deploying locally (e.g. in Minikube), it can be more efficient
+    to use a _local_ Docker registry.
```
Comment (arschles) [inline deploy/wip-catalog/README.md]: 👍

File: deploy/wip-catalog/README.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+- `insecure`: Whether the API server should serve insecurely; optional; defaults
+  to `false`
+- `insecurePort`: When `insecure` is true, the API server pod serves insecurely
+  on this container port; optional; defaults to `8081`
+- `insecureServicePort`: When `insecure` is true, the API server is fronted by a
+  k8s service serving on this node port; optional; defaults to `30001`
+- `storageType`: The type of storage for the API server to use; optional;
+  defaults to `etcd`
+- `globalNamespec`: The namespace to store global resources in when the API
+  server is backed by k8s Third Party Resources.
```
Comment (arschles) [inline deploy/wip-catalog/README.md]: what about `registry`, `k8sApiServerVersion`, `version`, `etcdRepository` , `version`?

File: deploy/wip-catalog/README.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+- `insecure`: Whether the API server should serve insecurely; optional; defaults
+  to `false`
+- `insecurePort`: When `insecure` is true, the API server pod serves insecurely
+  on this container port; optional; defaults to `8081`
+- `insecureServicePort`: When `insecure` is true, the API server is fronted by a
+  k8s service serving on this node port; optional; defaults to `30001`
+- `storageType`: The type of storage for the API server to use; optional;
+  defaults to `etcd`
+- `globalNamespec`: The namespace to store global resources in when the API
```
Comment (arschles) [inline deploy/wip-catalog/README.md]: s/globalNamespec/globalNamespace

File: deploy/wip-catalog/README.md (hunk 6 above)
Comment (arschles) [inline deploy/wip-catalog/README.md]: also, add a `controllerManagerVersion` -- maybe that should go into a new "Controller Manager Knobs" section?

**Comment by arschles** [comment]
I'd rather get this into master now and fix in follow-ups

**Pull request merged**